src/Features/LanguageServer/ProtocolUnitTests/References/FindAllReferencesHandlerTests.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.CodeAnalysis.LanguageServer.UnitTests.References
 {
     public class FindAllReferencesHandlerTests : AbstractLanguageServerProtocolTests
     {
-        [WpfFact]
+        [WpfFact(Skip = "https://github.com/dotnet/roslyn/issues/43063")]
```

Comment:
this is concerning :)  do we know why this is flakey?

Reply:
I made some last-minute test additions in the PR to move FAR to LSP, and those additions seem to be flaky. I plan to go back soon in a future PR and properly fix it up.